### DISCUSSION TITLE
Why Renovate GitHub App needs `Read access to emails` permission?

### DISCUSSION THREAD

**Discussion by dawidhyzy-ifolor** [renovatebot/renovate#14405] [Request Help] [answered]
I want to configure Renovate GitHub App for my organization but first, need information on why this permission is requested.

![Screenshot 2022-02-24 at 15 21 45](https://user-images.githubusercontent.com/77614220/155667485-58ebd8b3-8ef6-405b-b0f9-2b0b484ca908.png)

**Comment by rarkins** [comment, marked as answer]
User permissions are used only if you log into the app dashboard and authorise with oauth.   Ie you "sign in with GitHub" and in the process agree to the site terms, and your consent is stored against your email.  

This doesn't grant (as far as I'm aware) the app to automatically gather any email addresses, and nor do we.

**Reply by gauravsoni5555** [reply]
Gauravsoni3200@gmail.com
